tests/cli/test-migrate.sh

```
@@ -0,0 +1,99 @@
+
+# --- Test: does not overwrite existing tribunal.yaml ---
+TMPDIR2=$(mktemp -d)
+trap 'rm -rf "$TMPDIR" "$TMPDIR2"' EXIT
+
+echo "existing: true" > "$TMPDIR2/tribunal.yaml"
+mkdir -p "$TMPDIR2/.metaswarm"
+cat > "$TMPDIR2/.metaswarm/project-profile.json" << 'JSON'
+{"stack": "node"}
+JSON
+
+RESULT=$(node "$ROOT/lib/migrate-config.js" "$TMPDIR2" 2>&1)
```

Shellcheck (0.11.0)

[warning] RESULT appears unused. Verify use (or export if used externally).

(SC2034)